File: src/main/java/via/sep3/sep3_1/model/Recipe.java
package via.sep3.sep3_1.model;

public class Recipe
{
    private final String name;
    private final String[] ingridient;
    private final String description;

    public Recipe(String name, String[] ingridient, String description)
    {
        this.name = name;
        this.ingridient = ingridient;
        this.description = description;
    }

    public String[] getIngridient()
    {
        return ingridient;
    }

    public String getName()
    {
        return name;
    }

    public String getDescription()
    {
        return description;
    }
}

File: Tier1/SEP3_Tier1/BlazorAppTier1/CLient/Recipe.cs
namespace BlazorAppTier1.CLient;

public class Recipe
{
    private readonly string name;
    private readonly string[] ingridient;
    private readonly string description;

    public Recipe(string name, string[] ingridient, string description)
    {
        this.name = name;
        this.ingridient = ingridient;
        this.description = description;
    }

    public virtual string[] Ingridient
    {
        get
        {
            return ingridient;
        }
    }

    public virtual string Name
    {
        get
        {
            return name;
        }
    }

    public virtual string Description
    {
        get
        {
            return description;
        }
    }

    public override string ToString()
    {
        return "Recipe{" +
               "name='" + name + '\'' +
               ", ingridient=" + ingridient +
               ", description='" + description + '\'' +
               '}';
    }
}